### PR TITLE
add note in README

### PR DESCRIPTION
**Pull request by AntonTumasov**
Just to prevent people from failures

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -204,6 +204,7 @@ Optionally, you may supply an `acl_api_token`.  This will allow you to create
 ACLs if the anonymous token doesn't permit ACL changes (which is likely).
 The api token may be the master token, another management token, or any
 client token with sufficient privileges.
+### NOTE. If you're already using ACL tokens you should consider not to use HCL due to library using JSON.parse function to create/update tokens
```
Comment (solarkennedy) [inline README.md]: This reads a little funny. 

What does HCL have to do with this? And which json library, the one that puppet-consul uses?

Comment (AntonTumasov) [inline README.md]: It's related to those, who faced the same problem as we did. It doesn't work with beforehand set up ACL rules in Hashicorp DSL. And yes, JSON from your Ruby code.

Comment (solarkennedy) [inline README.md]: What was the actual result? Did puppet overwrite the tokens that were created with HCL?

Comment (AntonTumasov) [inline README.md]: Nope, it didn't. It just failed with parsing. That's why I added this note.

Comment (solarkennedy) [inline README.md]: How about

```
### NOTE: This module currently cannot parse ACL tokens generated through means other than this module. Don't mix Puppet and Non-puppet ACLs for best results!
```

Comment (AntonTumasov) [inline README.md]: If it suits you, do so.

Comment (solarkennedy) [inline README.md]: Will do.

**Pull request closed without merging**